build/teamcity/cockroach/coverage/roachtest_nightly_gce_impl.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+
+source "$dir/teamcity-support.sh"
+
+if [[ ! -f ~/.ssh/id_rsa.pub ]]; then
+  ssh-keygen -q -C "roachtest-nightly-bazel $(date)" -N "" -f ~/.ssh/id_rsa
+fi
+
+$root/build/teamcity/cockroach/nightlies/roachtest_compile_bits.sh --with-code-coverage amd64
```

Comment:
Might be worth a comment to emphasize that the hardcoded (target) architecture is _not_ actually preventing TC agent from running arm64, since it's going to be cross-built.